woocommerce/woocommerce-ios
# [Woo POS] Add WooCommerce POS framework

## Description


This adds a new app-level framework for WooCommerce POS.

This is intended to sit at the same level in the heirarchy as the WooCommerce target, i.e. it can import Yosemite or an equivalent framework. 

This new framework should contain UI and VMs for the Woo POS project, with as much business logic as possible in Yosemite or an equivalent level framework, isolating the view from details of networking or storage type concerns.

### Testing with imported frameworks
I have (separately) tested this making a network request, and importing that lower-level framework works well. Note that's not the intended approach, it's just a lot simpler to test by skipping out Yosemite, while conceptually it proves the same thing.

To do this test, I passed in the `Credentials` and `siteID`, and used them to create an `OrdersRemote` and make a GET request with it.

## Testing instructions


1. Launch the app
2. Navigate to Menu > Settings > Experimental features
3. Turn on the Woo POS toggle
4. Switch stores or restart the app
5. Navigate to Menu > Woo POS
6. Observe that the placeholder view is shown


---
